### ISSUE TITLE
Warning when I compile nominations_extended.sp

### ISSUE THREAD

**Issue by Klonken** [mbhound/sourcemod-mapchooser-extended#6]
Thank you for this amazing update! However, I get this warning when I compile this version of **nominations_extended.sp**:

// nominations_extended.sp(339) : warning 213: tag mismatch
// nominations_extended.sp(465) : warning 213: tag mismatch
// nominations_extended.sp(767) : warning 213: tag mismatch

Here is the line: **GetExcludeMapList(excludeMaps);**

Here is the native in the .inc file: **native void GetExcludeMapList(ArrayList array);**

Thanks in advance!

**Comment by mbhound** [comment]
The plugin should still compile with the warning. All it means is that there are spaces instead of tabs.

**Comment by Klonken** [comment]
> 
> 
> The plugin should still compile with the warning. All it means is that there are spaces instead of tabs.

Wouldn't it be **warning 217: loose indentation** then?

**Comment by mbhound** [comment]
Yeah you're right I misread it. It probably doesn't like the type of the argument that is getting passed but it should still work. This was from the original code so I haven't really looked at it. I'll see if I can fix it when I have some time.
